### PR TITLE
Updating Testflinger agents for the following 10  servers:

### PR DESCRIPTION
**Pull request by mreed8855**
I have made updates for the following servers for testflinger
snide
serviceberry
salout
saiph
rozary
romano
rockman
polari
pikmin
pianta

### CODE REVIEW

File: sut/pikmin_snappy.yaml
Comment (bladernr) [inline sut/pikmin_snappy.yaml:1]: this one is missing default storage information....

**Review by bladernr — CHANGES_REQUESTED** [review]
ONe of them is missing the storage info... otherwise looks fine.  Fix that one and we can merge this.  Don't forget to check them off on the spreadsheet if you haven't done so already.

File: sut/pikmin_snappy.yaml
Comment (mreed8855) [reply sut/pikmin_snappy.yaml:1]: I have moved the pikmin files back to the UPDATES_NEEDED directory as that system has network issues that are preventing it from commissioning and updating the storage configuration.

**Review by bladernr — APPROVED** [review]
Thanks!

**Pull request merged**